### ISSUE TITLE
[feat] testnet LNbits environment (testnet.lnbits.com)

### ISSUE THREAD

**Issue by motorina0** [lnbits/lnbits#2483]
Always runs `dev`.
For testing & regression purposes.

testnet.lnbits.com

**Comment by prusnak** [comment]
maybe signet is better idea since testnet is being DoSed by lopp?

**Comment by bitkarrot** [comment]
please run both. testnet has been cripped lately, if we have a signet option it will be great

a signet faucet would also be handy close by

**Comment by dni** [comment]
i agree on a testnet lnbits! it could be useful for developers working on their own apps making testnet more accessable

**Comment by bitkarrot** [comment]
If you guys don't get around to running this that's also OK, we'll be running lnbits testnet over at plebnet.dev for the next few years at a minimum https://plebnet.dev/testnet_how_to

**Comment by dentropy** [comment]
I just spent a couple hours trying to get lnbits working with simnet, I got no luck, if anyone has any docs on getting that running I would love to take a look at it

For context I am using LND working and sucesfully followed this tutorial,

https://github.com/lightningnetwork/lnd/tree/master/docker
